helm/trino-gateway/templates/deployment.yaml

```diff
@@ -43,12 +43,8 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command:
-              - "java"
-              - "-XX:MinRAMPercentage=80.0"
-              - "-XX:MaxRAMPercentage=80.0"
-              - "-jar"
-              - "/usr/lib/trino/gateway-ha-jar-with-dependencies.jar"
-              - "/etc/gateway/config.yaml"
+              - java
+              {{- toYaml append .Values.javaOptions list "-XX:MinRAMPercentage=80.0" "-XX:MaxRAMPercentage=80.0" "-jar" "/usr/lib/trino/gateway-ha-jar-with-dependencies.jar" "/etc/gateway/config.yaml"}}
```

Comment:
How about moving the options all into the values file

Reply:
In this setup you cant really override those values right? Or at least they would show up twice..

Reply:
that could be nice and simple, i'll update this